contracts/PixieCrowdsale.sol

```diff
@@ -179,7 +179,7 @@ contract PixieCrowdsale is Crowdsale, Pausable {
    * @param _privateSaleCloseTime the epoch time to set
    */
   function updatePrivateSaleCloseTime(uint256 _privateSaleCloseTime) external onlyManagement {
-    require(_privateSaleCloseTime > 0, "A private sale time must be specified");
+    require(_privateSaleCloseTime > openingTime, "A private sale time must after the opening time");
```

Comment:
Are we just adding logic here for the sake of it? The previous validation was a non-null check and we've replaced it with a business logic check.

Reply:
it's still non-null but better?